src/tigerbeetle/cli.zig

```diff
@@ -493,7 +493,7 @@ pub fn parse_args(allocator: std.mem.Allocator, args_iterator: *std.process.ArgI
             if (pipeline_limit < pipeline_limit_min) {
                 flags.fatal("--limit-pipeline-requests: count {} is below minimum: {}", .{
                     pipeline_limit,
-                    pipeline_limit_min,
+                    vsr.stdx.fmt_int_size_bin_exact(pipeline_limit_min),
```

Comment:
This is a count of message, not a size in bytes, so I think should be left as is

(this might be a dulicate comment, GitHub is acting weirdly for me here)